stories/Colors.stories.mdx

```diff
@@ -0,0 +1,187 @@
+import { Meta, ColorPalette, ColorItem } from '@storybook/addon-docs';
+
+<Meta title="Base/Colors" />
+
+# Colors
+
+## Global
+
+<ColorPalette>
+  <ColorItem
+    title="Base"
+    colors={{
+      'base-0': '#ffffff',
```

Comment:
Should these colours be pulled from common file? It seems likely they'll drift out from reality which would make this page not so useful. 

Maybe there's a way to import JSON to scss?